addons/website_slides/tests/test_ui_wslides.py

```diff
@@ -331,6 +331,7 @@ def test_course_member_yt_employee(self):
             'group_ids': [(5, 0), (4, self.env.ref('base.group_user').id)]
         })
         self.env.ref('website_slides.slide_channel_demo_3_furn0')._remove_membership(self.env.ref('base.partner_demo').ids)
+        self.env.ref('website_slides.slide_slide_demo_3_1').url += '&start=260'
```

Comment:
What is this? What is 260?

Reply:
It's to go directly to the end of the video and have the slide marked as done